samza-core/src/main/java/org/apache/samza/storage/TaskSideInputStorageManager.java

```diff
@@ -184,6 +188,14 @@ public String getLastProcessedOffset(SystemStreamPartition ssp) {
     return lastProcessedOffsets.get(ssp);
   }
 
+  /**
+   * For unit testing
+   */
+  @VisibleForTesting
+  void updatelastProcessedOffset(SystemStreamPartition ssp, String offset) {
```

Comment:
`updateLastProcessedOffset`
"For unit testing only."
Move to bottom of class, or after getLastProcessedOffset if it exists.

Reply:
It is already under `getLastProcessedOffset`

Reply:
Minor: Need to fix the method name too (L in Last should be capital).